### PR TITLE
movie update added

### PR DESCRIPTION
**Pull request by fkgdmk**
kom i gang rondo

### CODE REVIEW

File: src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java [hunk 1]
```diff
@@ -9,6 +9,8 @@
 import java.util.ArrayList;
 import java.util.List;
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java:11]: remove spacing

**Review by AnderskdKnudsen — COMMENTED** [review]
Good job hobos

File: src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java (hunk 1 above)
Comment (fkgdmk) [reply src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java:11]: done

Comment (fkgdmk) [reply src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java:11]: outdated

**Review by AnderskdKnudsen — APPROVED** [review]
good jb

**Pull request merged**